### ISSUE TITLE
Fails to build in Xcode-13 Beta for iOS 15

### ISSUE THREAD

**Issue by Crizzooo** [fullstorydev/fullstory-react-native#27]
Hi,

We are testing our app on the iOS 15 beta to ensure we are ready.

In new Xcode-13 beta, we are unable to play/archive our build. A fullstory error fails the process with

```
error: FullStory: [2021-08-30 11:37:38:515] FullStoryCommandLine(34955:369180) Failed to run asset extraction with uncaught signal 4: Illegal instruction
error: FullStory: [2021-08-30 11:37:38:515] FullStoryCommandLine(34955:369180) Failed to extract CUIAssets using simulator.
```

Is this a known issue?

Will it be fixed prior to the release of iOS 15 later this month?

**Comment by jwise** [comment]
Hi Christopher --

Oof, this looks like a bug.  We're not really set up to handle support requests via GitHub, so even though this will end up getting escalated back to me, would you mind sending some mail to our support team at mobile-support@fullstory.com to make sure that I get an internal ticket created for it?  In your e-mail, please include the build number of the Xcode-13 beta that you're using, and whether you're on Intel or Apple Silicon.

Thanks --
joshua

**Comment by jwise** [comment]
I just checked the support queue, and it looks like Elizabeth should have a solution that gets you up and running -- I also think this was fixed in 1.17.0, so updating to the latest plugin release should help you out.  I'll close this for now; feel free to reach out to support if you need anything else!

**Comment by CarolynWebster** [comment]
What was the proposed fix for this? If we upgrade to 1.17.0 it breaks our app for anyone using iOS 12.4

**Comment by jwise** [comment]
Hi Carolyn -- can you send mail to mobile-support@fullstory.com ?  I just flipped through Zendesk and found a ticket discussing this previously, and we've had a lot of improvements since the version you're using; we'd like to make sure we can get you upgraded!
